tools/protoxform/protoxform.py

```diff
@@ -163,24 +167,23 @@ def CamelCase(s):
   package_line = 'package %s;\n' % file_proto.package
   file_block = '\n'.join(['syntax = "proto3";\n', package_line])
 
-  options = [
-      'option java_outer_classname = "%s";' % CamelCase(os.path.basename(file_proto.name)),
-      'option java_multiple_files = true;',
-      'option java_package = "io.envoyproxy.%s";' % file_proto.package,
-  ]
+  options = file_proto.options
```

Comment:
Thanks for the update.

I actually consider what we today in terms of enforcing a strict set of options to be a feature not a bug. I think we should avoid having folks doing random language-specific options, which historically has resulted in a lot of inconsistency.

I do like the move away from string templates to using first class proto though.

I think we should whitelist specific options that folks want to add here, and erase others.

Reply:
OK, let's start from an empty FileOptions then, now it should match string template.